recipes/_efa_config.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: _efa_config
```

Comment:
AFAIK we are getting rid of the underscore prefix for the recipes.
If so I'd remove it from here and from the _efa_install too.

Reply:
done in a separate commit